code/modules/events/ghost_role/abductor.dm

```diff
@@ -20,8 +20,13 @@
 		return NOT_ENOUGH_PLAYERS
 
 	SSmapping.lazy_load_template(LAZY_TEMPLATE_KEY_ABDUCTOR_SHIPS)
-	var/mob/living/carbon/human/agent = make_body(pick_n_take(candidates))
-	var/mob/living/carbon/human/scientist = make_body(pick_n_take(candidates))
+	// BUBBER CHANGE: NO ABDUCTOR PREFS
+	var/mob/living/carbon/human/agent = new
+	var/mob/living/carbon/human/scientist = new
+
+	agent.PossessByPlayer(candidates[1].ckey)
```

Comment:
where does this end up putting them? make_body also contains code to move the mob, did you test if it puts them anywhere?

Reply:
I tested it, theyre in nullspace until the abductor antag datum brings them to the ship